__mocks__/@auth0/auth0-react.js

```diff
@@ -0,0 +1,8 @@
+const useAuth0 = () => ({
+  user: "random user",
```

Comment:
Can we update this `user` object to also include fields for
`email_verified`
`email`
`https://cruzhacks.com/roles`
`sub`

Reply:
Along with this, have it so email_verified can either be tested with true or false. https://cruzhacks.com/roles will have an array of values either including the value "Organizer" or "User"

Reply:
Sorry, I'm kind of confused for the third field, from what I'm reading the object should look like this:
`{
  email_verified: true/false,
  email: "mock email",
  roles: ["Organizer", "User"],
  sub: "mock user"
}`

Correct me if I'm wrong here

Reply:
What you have is correct, but roles will only contain Organizer or User. Auth0 stores roles as an array within the idToken.

Reply:
@sidhantas One other thing to correct is that the title of the field is "https://cruzhacks.com/roles"

Reply:
Ah, I got it, just had to put the token into jwt.io to figure it out